src/main/java/com/codefactorygroup/betting/service/CompetitionServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.codefactorygroup.betting.service;
+
+import com.codefactorygroup.betting.converter.CompetitionDTOtoCompetitionConverter;
+import com.codefactorygroup.betting.converter.EventDTOtoEventConverter;
+import com.codefactorygroup.betting.domain.Competition;
+import com.codefactorygroup.betting.dto.CompetitionDTO;
+import com.codefactorygroup.betting.repository.CompetitionRepository;
+import org.springframework.stereotype.Service;
+
+@Service(value = "competitionService")
+public class CompetitionServiceImpl implements CompetitionService{
+
+    private final CompetitionRepository competitionRepository;
+    private final CompetitionDTOtoCompetitionConverter competitionDTOtoCompetitionConverter;
+
+    private final EventDTOtoEventConverter eventDTOtoEventConverter;
+
+    public CompetitionServiceImpl(CompetitionRepository competitionRepository, CompetitionDTOtoCompetitionConverter competitionDTOtoCompetitionConverter, EventDTOtoEventConverter eventDTOtoEventConverter) {
+        this.competitionRepository = competitionRepository;
+        this.competitionDTOtoCompetitionConverter = competitionDTOtoCompetitionConverter;
+        this.eventDTOtoEventConverter = eventDTOtoEventConverter;
+    }
+
+    @Override
```

Comment:
Use @Transactional annotation over the service methods that execute DB operations